src/core/include/openvino/core/runtime_attribute.hpp

```diff
@@ -31,6 +31,7 @@ class OPENVINO_API RuntimeAttribute {
     virtual ~RuntimeAttribute() = default;
     virtual bool is_copyable() const;
     virtual bool is_copyable(const std::shared_ptr<Node>& to) const;
+    virtual bool is_copyable(const std::shared_ptr<Node>& from, const std::shared_ptr<Node>& to) const;
```

Comment:
Can we avoid adding this member to public API, it looks like is required only by one attribute, others don't need it.
Would be better to have specific solution unless this member can give benefits for others attributes.